blazor/gantt-chart/zooming.md

```diff
@@ -134,3 +134,62 @@ In the Gantt chart, the zoom-in and zoom-out actions are performed based on the
 ```
 
 ![Customizing zooming levels in Blazor Gantt Chart](images/blazor-gantt-chart-custom-zooming-levels.gif)
+
+## Resetting zooming levels using method
```

Comment:
Resetting Zoom Levels Using Method

In the Gantt chart, you can reset the zoom level to its initial state, as configured during the initial rendering, after performing zooming actions such as ZoomIn, ZoomOut, and ZoomToFit, using the [ResetZoomAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_ResetZoomAsync) method.